cli/tests/integration_tests.rs

```diff
@@ -5969,4 +5972,7 @@ async fn listen_tls_alpn_fail() {
   let (_, session) = tls_stream.get_ref();
 
   assert!(session.get_alpn_protocol().is_none());
+
+  child.kill().unwrap();
+  child.wait().unwrap();
```

Comment:
Is this demonstrating the bug?

Reply:
It prevents the test from leaking a child process. Although this only happens when it fails (otherwise it exits by itself), the  leaked process subsequently squats a TCP listener port and makes a bunch of other tests fail, which makes it hard to debug.